openfisca_us/variables/demographic/person.py

```diff
@@ -112,3 +112,55 @@ class provider_type_group(Variable):
     entity = Person
     label = u"CCDF provider type group"
     definition_period = YEAR
+
+
+class ccdf_hours_per_week(Variable):
+    value_type = float
+    entity = Person
+    label = u"Hours per week"
+    definition_period = YEAR
+
+
+class ccdf_hours_per_day(Variable):
+    value_type = float
+    entity = Person
+    label = u"Hours per day"
+    definition_period = YEAR
+
+
+class DurationOfCare(Enum):
+    WEEKLY = "Weekly"
+    DAILY = "Daily"
+    PART_DAY = "Part-Day"
+    HOURLY = "Hourly"
+
+
+class duration_of_care(Variable):
+    value_type = Enum
+    possible_values = DurationOfCare
+    default_value = DurationOfCare.WEEKLY
+    entity = Person
+    label = u"CCDF duration of care"
+    definition_period = YEAR
+
+    reference = "https://ocfs.ny.gov/main/policies/external/ocfs_2019/LCM/19-OCFS-LCM-23.pdf"
+
+    def formula(person, period):
+        hours_per_week = person("ccdf_hours_per_week", period)
+        hours_per_day = person("ccdf_hours_per_day", period)
+        return select(
+            [
+                hours_per_week >= 30,
```

Comment:
@MaxGhenis I would actually be fine with not having these as parameters, but WDYT? Could do for consistency

Reply:
I think this is OK for now, though other states might have different values so we may have to parameterize later.